tests/shared/core/training_data/test_visualization.py

```diff
@@ -142,9 +142,9 @@ def test_story_visualization(domain: Domain, tmp_path: Path):
     )
 
     assert str(None) not in out_file.read_text()
-    assert "/affirm" in out_file.read_text()
-    assert len(generated_graph.nodes()) == 51
-    assert len(generated_graph.edges()) == 56
+    assert "affirm" in out_file.read_text()
+    assert len(generated_graph.nodes()) == 53
+    assert len(generated_graph.edges()) == 58
```

Comment:
I'm not familiar with these unit tests, so a quick explanation on why the changes in affected unit tests were required would be helpful 🙏🏻

Reply:
For the tests, it is quite simple, we have new graph nodes and edges now because we are adding slots onto the visualization, which wasn't the case previously.